modules/lib/middleware.js

```diff
@@ -9,7 +9,7 @@ const _verify = ({
   rawBody,
 }) => {
   const hmac = crypto.createHmac('sha256', SLACK_SIGNING_SECRET)
-  const [version, hash] = signature.split('=')
+  const [version, hash] = (signature || '').split('=')
```

Comment:
Should an error be thrown here instead (i.e. does it make sense for `_verify` to go on with `version` and `hash` as `''`?)?

Reply:
That was the previous behaviour, which would lead to a bit harder debugging experience if it was actually errored. but an error response to slack isn't supported

Though it makes sense to skip the computation below if both `version` and `hash` are empty, in which case we can simply return a `false` to save a bit of trees

Reply:
also the next step is to move us to use the official `bolt.js`, which has all these verifications abstracted away, so we don't risk them changing these things at slack side